verilog/analysis/checkers/no_trailing_spaces_rule.cc

```diff
@@ -53,13 +53,18 @@ const LintRuleDescriptor& NoTrailingSpacesRule::GetDescriptor() {
 }
 
 void NoTrailingSpacesRule::HandleLine(absl::string_view line) {
+  // Lines may end with \n or \r\n. '\n' is already excluded.
+  // Exclude '\r'
+  if (!line.empty() && line.back() == '\r') {
+    line.remove_suffix(1);
```

Comment:
Check out the functionality provided by absl. This looks like a good canidate for `absl::ConsumeSuffix(&line, "\r")`

Reply:
Thank you! Good to know, especially that `ends_with` from `std` comes in C++20, so It wouldn't fit here.
Right now this code uses `absl::ConsumeSuffix`.